### PR TITLE
Gradle added

### PR DESCRIPTION
**Pull request by colinm0125**
Created a new file with all of the Gradle files downloaded from the Brightspace as instructed. Also pasted the Gradle yml file into the workflows alongside the checkstyle.

### CODE REVIEW

**Comment by colinm0125** [comment]
Cannot access Gradlew, will look at it later, I need to charge my laptop

**Comment by babbo65** [comment]
chmod command running in wrong directory, added command to yml to cd into correct directory

File: .github/workflows/gradle.yml [hunk 1]
```diff
@@ -0,0 +1,37 @@
...
+        java-version: '17'
+        distribution: 'temurin'
+
+    # Configure Gradle for optimal use in GiHub Actions, including caching of downloaded dependencies.
+    # See: https://github.com/gradle/actions/blob/main/setup-gradle/README.md
+    - name: Setup Gradle
+      uses: gradle/actions/setup-gradle@417ae3ccd767c252f5661f1ace9f835f9654f2b5 # v3.1.0
+
+    - name: Build with Gradle Wrapper
+      run: cd gradle; chmod a+rwx gradlew; ./gradlew build
```
Comment (babbo65) [inline .github/workflows/gradle.yml:37]: added cd gradle, then let it execute other commands

**Review by babbo65 — APPROVED** [review]
Looks great! Thank you Colin McDermott.

**Pull request merged**